scripts/generate_swagger.sh

```diff
@@ -1,19 +1,17 @@
 #!/usr/bin/env bash
 
+set -e
+
 CONVERT_URL="https://converter.swagger.io/api/convert"
-DOCS_TMP="/tmp"
+DOCS_DIR="./manager/docs"
 
 if ! which "$GOPATH"/bin/swag &> /dev/null; then
     go install github.com/swaggo/swag/cmd/swag@latest
 fi
 
-# Generate 2.0 swagger
-"$GOPATH"/bin/swag init -g ./manager/manager.go --output $DOCS_TMP
-
-if ! ls ./manager/docs &> /dev/null; then
-  mkdir ./manager/docs
+if ! ls "$DOCS_DIR" &> /dev/null; then
+  mkdir "$DOCS_DIR"
 fi
 
-# Convert 2.0 -> 3.0
-curl -X POST -H "Accept: application/json" -H "Content-Type: application/json" \
-     -d @$DOCS_TMP/swagger.json $CONVERT_URL > ./manager/docs/swagger.json
+# Generate 2.0 swagger
```

Comment:
Why not converting it to 3.0?

Reply:
Because it doesn't convert correctly - references are broken, clicking around the swagger UI is throwing errors and structure examples are wrong. I found out that the converter wasn't updated for few years, if we want swagger 3, we need to find different way.

Reply:
Thanks